packages/calcite-components/src/components/tip-manager/tip-manager.scss

```diff
@@ -1,24 +1,35 @@
+@import "../../assets/styles/header";
+
 /**
- * CSS Custom Properties
- *
- * These properties can be overridden using the component's tag as selector.
- *
- * @prop --calcite-tip-manager-height: The maximum height of the component.
- * @prop --calcite-tip-max-width: The maximum width of a slotted `calcite-tip` within the component.
- */
+* CSS Custom Properties
+*
+* These properties can be overridden using the component's tag as selector.
+*
+* @prop --calcite-tip-max-width: [Deprecated] No longer necessary. 
+* @prop --calcite-tip-manager-height: [Deprecated] No longer necessary. 
+* @prop --calcite-tip-manager-background-color: The background color of the component.
+* @prop --calcite-tip-manager-heading-text-color: The heading text color of the component.
+* @prop --calcite-tip-manager-text-color: The text color of the component.
+* @prop --calcite-tip-manager-border-color: The border color of the component.
+* 
+* @prop --calcite-tip-manager-tip-border-color: The border color of the sub-component.
```

Comment:
I don't think these tip-specific props are applicable. Tips are slotted into the manager and are not internal.